ui/src/scss/_buttons.scss

```diff
@@ -76,7 +76,7 @@ fieldset:disabled a.btn {
   text-shadow: 0 1px 1px #0c862f;
   @extend .rb-medium;
 
-  &:hover, &:active, &:focus {
+  &:hover{
     background-color: #039465 !important;
```

Comment:
plz use color variables instead of hex code

Reply:
bro its in scss file, we always declare color variables in scss file right?

Reply:
made changes for variables